Formula/opencv.rb

```diff
@@ -27,14 +27,14 @@ class Opencv < Formula
   depends_on "ffmpeg@4"
```

Comment:
Note to check on `ffmpeg@4 --> ffmpeg` too: https://github.com/opencv/opencv/pull/21754